Monika After Story/game/script-topics.rpy

```diff
@@ -15749,3 +15749,41 @@ label monika_literature_value:
     m 1eku "Honestly, I think if more people valued books and poems a little more, the world would be a much better place."
     m 1hksdlb "That's just my opinion as president of a literature club, though. {w=0.2}I guess most people wouldn't think that deeply about it."
     return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_renewable_energy",
+            category=['technology'],
+            prompt="Renewable energy",
+            random=True
+        )
+    )
+
+label monika_renewable_energy:
+    m 4eub "What do you think about renewable energy, [player]?"
+    m 4eud "It's a complicated subject; we talked about it a lot in the Debate Club."
```

Comment:
Debate club isn't capitalized, for consistency. 
```suggestion
    m 4eud "It's a complicated subject; we talked about it a lot in the debate club."
```

Reply:
I've seen people capitalize the literature club as 'Literature Club', so is that the exception to the rule or are those punctuation errors as well?